### DISCUSSION TITLE
Frame Data Analysis Graph with Hex Numbers?

### DISCUSSION THREAD

**Discussion by FrankM54** [collin80/SavvyCAN#412] [Ideas]
(Collin80: Thanks for all you do in creating and supporting the wonderful tool. I greatly appreciate it!!)

I use the Frame Data Analysis window a lot, and it would be nice to be able to switch the graph view between decimal and hex numbers.

Is there any change this could be added?

**Comment by collin80** [comment]
I suppose that could be a preference that you can set. There are other places where this could be set too. The FlowView screen also has a graph of each byte and it is currently 0-255 not 0-FF. For either one it could be useful to see the values in HEX instead. I will work on adding that.
